databases.go

```diff
@@ -589,6 +589,9 @@ type PostgreSQLConfig struct {
 	BackupMinute                    *int                         `json:"backup_minute,omitempty"`
 	WorkMem                         *int                         `json:"work_mem,omitempty"`
 	TimeScaleDB                     *PostgreSQLTimeScaleDBConfig `json:"timescaledb,omitempty"`
+	SynchronousReplication          *string                      `json:"synchronous_replication,omitempty"`
+	StatMonitorEnable               *bool                        `json:"stat_monitor_enable,omitempty"`
+	MaxFailoverReplicationTimeLag   *uint64                      `json:"max_failover_replication_time_lag,omitempty"`
```

Comment:
Could `MaxFailoverReplicationTimeLag` be reduced to `*int64` max - `9223372036854775807`?

Currently, [API doc](https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_get_config) is showing `9223372036854776000` as a max. This value can be because of [JS rounding](https://stackoverflow.com/questions/15529337/prevent-javascript-number-function-from-rounding-big-numbers) and in real life. Do you know if we can safely reduce it to int64 `9223372036854775807`?

Reply:
The field MaxFailoverReplicationTimeLag should remain as *uint64 for the following reasons:

Ensuring Non-Negative Values: The uint64 type guarantees that only non-negative values can be stored, which aligns with the requirement that MaxFailoverReplicationTimeLag must always be a positive number (as per the API documentation). Using *int64 introduces the possibility of negative values, which would require additional validation to prevent, whereas uint64 inherently enforces this constraint.

Valid API Range: The range defined in the API documentation ([10 .. 9223372036854776000]) falls within the capabilities of uint64, which can store values from 0 to 18446744073709551615. This ensures that the full range of valid values can be safely represented.

Avoiding Redundant Validation: If we switch to *int64, we'd need extra checks to ensure the values are non-negative, adding unnecessary complexity. *uint64 is semantically correct because we are working with a positive range of numbers, and the type itself enforces this rule.

Precision and Consistency: The value 9223372036854776000 in the API documentation may be a result of JavaScript rounding, and it is just slightly over the maximum value of int64. Using *uint64 accommodates this slight discrepancy without risk of truncation or loss of data precision.